libs/java/auth_core/src/test/java/com/yahoo/athenz/auth/token/jwts/JwtsHelperTest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Verizon Media
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.athenz.auth.token.jwts;
+
+import org.testng.annotations.Test;
+import static org.testng.Assert.*;
+
+public class JwtsHelperTest {
+
+    @Test
+    public void testExtractJwksUri() {
+
+        MockJwtsHelper.setResponseCode(200);
+        MockJwtsHelper.setResponseBody("{\"token_endpoint\":\"https://localhost/oauth2/token\",\"jwks_uri\":\"https://localhost/oauth2/keys\"}");
```

Comment:
Is it possible for the attribute jwks_uri to exist but be empty? Or exist but be malformed (not in a URI structure)? If so we should probably add these test cases

Reply:
added test case for null/empty uri. malformed - library doesn't care. it's up to the caller to verify if it's valid or not.